dotnet/src/SemanticKernel.Abstractions/AI/ChatCompletion/IChatStreamingResult.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using System.Threading;
+
+namespace Microsoft.SemanticKernel.AI.ChatCompletion;
+
+public interface IChatStreamingResult : IChatResult
+{
+    /// <summary>
+    /// Get the chat message from the streaming result.
+    /// </summary>
+    /// <param name="cancellationToken">The <see cref="CancellationToken"/> to monitor for cancellation requests. The default is <see cref="CancellationToken.None"/>.</param>
+    /// <returns>Current chat message streaming content</returns>
+    IAsyncEnumerable<ChatMessageBase> GetChatMessageStreamingAsync(CancellationToken cancellationToken = default);
```

Comment:
Should this not also be in the format "GetStreaming<<things>>Async"?
`GetStreamingChatMessageAsync`